test/asyncapi_test.go

```diff
@@ -40,6 +40,37 @@ func (s *CLITestSuite) TestAsyncApiExport() {
 			s.Error(nil, "spec generated does not match the template output file")
 		}
 	}
+	resetConfiguration(s.T(), false)
+}
+
+func (s *CLITestSuite) TestAsyncApiImport() {
+	tests := []CLITest{
+		{args: "asyncapi import", exitCode: 1, fixture: "asyncapi/err_no_file.golden"},
+		{args: "asyncapi import ./test/fixtures/input/asyncapi/asyncapi-spec.yaml", exitCode: 1, fixture: "asyncapi/no_kafka.golden"},
+		{args: "asyncapi import ./test/fixtures/input/asyncapi/asyncapi-spec.yaml", exitCode: 1, useKafka: "lkc-asyncapi", authKafka: "true", fixture: "asyncapi/no_sr_key.golden"},
+	}
+	resetConfiguration(s.T(), false)
+	for _, test := range tests {
+		test.login = "cloud"
+		s.runIntegrationTest(test)
+	}
+	resetConfiguration(s.T(), false)
```

Comment:
No need to reset the configuration at the end. Other suites will do this at the start if necessary.
```suggestion
```